src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/noc/nocworkflows/UpdateRepresentationService.java

```diff
@@ -98,7 +101,7 @@ public Map<String, Object> updateRepresentationAsSolicitor(CaseDetails caseDetai
 
     private UserDetails getInvokerDetails(String authToken, CaseDetails caseDetails) {
         AuditEvent auditEvent = auditEventService.getLatestAuditEventByName(caseDetails.getId().toString(), NOC_EVENT)
-            .orElseThrow(() -> new IllegalStateException(String.format("Could not find %s event in audit", NOC_EVENT)));
+            .orElseThrow(() -> new IllegalStateException(format("Could not find %s event in audit", NOC_EVENT)));
```

Comment:
Add the case id to the exception message

Reply:
[f726c09](https://github.com/hmcts/finrem-case-orchestration-service/pull/1781/commits/f726c09f63e9072141d54c68c3318ca19ff0c6b5)